src/utils/cookieJar.js

```diff
@@ -12,4 +12,9 @@ governing permissions and limitations under the License.
 
 import cookie from "@adobe/reactor-cookie";
 
-export default cookie;
+ export const cookieJarWithEncoding = cookie.withConverter({
+  write: (value) => {
+    return encodeURIComponent(value);
```

Comment:
I don't understand how this is different than the default `js-cookie` behavior, since they both use `encodeURIComponent()`. 
Here is the default converter from `js-cookie`, for reference

```js
function write (value) {
  return encodeURIComponent(value).replace(
    /%(2[346BF]|3[AC-F]|40|5[BDE]|60|7[BCD])/g,
    decodeURIComponent
  )
}
``` 
Is the difference that we DO need to encode special characters like # and $ (which is what that regex does-it matches the URI-encoded codes for special characters)?

Reply:
Ignore that comment. I read the ticket description. The answer is "yes, we do need to encode everything all of the time".